src/Components/Book/Book.js

```diff
@@ -3,13 +3,16 @@ import ReactDOM from 'react-dom';
 import './Book.css'
 
 
-const Book = ({book}) => {
+const Book = ({ book, addBook }) => {
   return (
     <section class="book-card-section">
         <h1>{book.title}</h1>
         <h3>{book.author}</h3>
         <h3>Ranking: {book.rank}</h3>
         <img className="Book-card-image" alt="Book cover" src={book.book_image} />
+        <button id={book.primary_isbn10} onClick={addBook} className="reading-list-button active">
+          Reading List
```

Comment:
Make we make the text for this button an action like "Add to Reading List" or "Read"? I used "Add to Reading List" on the book info page, so maybe "Read" would work. Do you think a user would understand what the button does if it just said "Read"?

Reply:
Completely agreed!!